.travis.yml

```diff
@@ -170,6 +184,7 @@ addons:
 
 env:
   global:
+    - RUN_DROP_TRAVIS_CACHES_STAGE=true
```

Comment:
so always drop caches, not manually triggering?

Reply:
Yes, we saw in akka-platform-guide that this hits PR branches which doesn't allow manual Travis triggers.

Reply:
gotcha